components/web/ejabberd/Makefile

```diff
@@ -45,10 +45,14 @@ include $(WS_MAKE_RULES)/prep.mk
 include $(WS_MAKE_RULES)/configure.mk
 include $(WS_MAKE_RULES)/ips.mk
 
-CC=gcc -m64 -I/usr/include/gd2
-CPP=gcc -m64 -E -I/usr/include/gd2
-CXX=g++ -m64 -I/usr/include/gd2
-LDFLAGS += -L/usr/lib/libjpeg8-turbo/lib/amd64 -R/usr/lib/libjpeg8-turbo/lib/amd64 -lnsl -lsocket
+CC=gcc
+CPP=gcc -E
+CXX=g++ -m64
```

Comment:
Why do we need this?

Reply:
I didn't figure out how to avoid this definitions.
CC=gcc
CPP=gcc -E
CXX=g++ -m64
If not used rebar compiles *.cpp as 32 bit.
And uses cc instead of gcc.

Reply:
At least $(CC_BITS) would be preferred.

Reply:
The next commit solves the problem.